demo_plugin/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "demo_plugin"
-version = "0.1.0"
+version = "0.1.5"
```

Comment:
Do we need to jump to 5?

Reply:
For all of the plugins to match newhelm's version, yep. Is this a problem? I can manually delete packages in the repo and start back at 0.1.0 if there's a reason to do that, but the version numbers are going to have to be incremented every time there's a new package to publish, so these will be going up a lot, arbitrarily.

Reply:
It's not a problem, just wanted to confirm it wasn't some accidental thing where you tested bumping it a few times.